### ISSUE TITLE
New device connection alert no longer availabe in >5.0 (Device connection/disconnection script)

### ISSUE THREAD

**Issue by 3freet** [ntop/ntopng#6636]
[#4709](https://github.com/ntop/ntopng/issues/4709) is when the feature was first introduced.

Also the Device connection/disconnection script no longer included on newly installed versions. And when trying to add it manually, it breaks many things.

If #6192 is meant to alert on newly detected devices that had never been seen on the network before, just like #4709 did, then this issue might be a duplicate.

**Comment by MatteoBiscosi** [comment]
Hi @3freet we decided to move that alert to 'Unexpected device connected'.
Here you can add a list of excluded Devices (MACs). Whenever a new Device is connected to the network for the first time and it's not part of the exclusion list then the alert is going to be triggered. 
![Screenshot from 2022-05-30 18-09-26](https://user-images.githubusercontent.com/49585191/171029516-80f9dad7-64e5-4332-ab74-6de5fc94a410.png)
![Screenshot from 2022-05-30 18-12-42](https://user-images.githubusercontent.com/49585191/171030076-70426120-73b7-4e9f-ba55-a0b0fb0a7e5a.png)

**Comment by 3freet** [comment]
Is there a way to add all newly connected devices to this list automatically so that we still get alerted but only once?
